test/Eto.Test/Eto.Test.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard1.6</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
I pondered this, but decided to keep Eto.Test as net standard 1.6.  It reduces the required dependencies so you don't have to install .net core sdk to compile everything.  That being said, the new gtk platform will require it so maybe that's just better.  Just a brain dump, let me know if you feel strongly either way.

Reply:
This was a mistake on my part XD